.github/workflows/bcny-firebase.yml

```diff
@@ -137,6 +156,24 @@ jobs:
             Copy-Item -Path $library.FullName -Destination $destination -Force
             Write-Host "... copied ${destination}"
           }
+
+      # We need this library to be used on CPUs without BMI2 support, so we check that snappy.lib was built correctly,
+      # and fail if it contains BMI2 instructions.
+      - name: Check for Snappy BMI2 instructions
+        # BMI2 instructions are only relevant in amd64
+        if: ${{ matrix.arch }} == 'amd64'
+        run: |
+          Write-Host "Checking for BMI2 instructions on Snappy.lib..."
+          $snappy_lib = "${{ github.workspace }}/BuildRoot/Library/firebase/usr/libs/windows/snappy.lib"
+          $output = dumpbin /DISASM "$snappy_lib" | Select-String -Pattern "bzhi"
```

Comment:
I really don't like this approach. We should use `llvm-objdump` and disable the bmi2 extensions and identify an unknown instructions.

Reply:
Do you have more specific `llvm-objdump` command line flags or settings? When I `llvm-objdump` a local build of snappy.lib I see a lot of `<unknown>` already. If you have a suggestion on how to do this, I would really appreciate since time is really tight to get this into this week's release.

Reply:
As discussed by DM, there was a lack of clarity on my code comment above and it did not explain why I was explicitly checking for `bzhi`. I added more context in the latest commit, PTAL.